package.json

```diff
@@ -86,6 +86,7 @@
         "client": "tsx scripts/cli.ts client"
     },
     "dependencies": {
+        "@hono/node-server": "^1.19.7",
```

Comment:
If this is a dependency for the example only shouldn't this be a devDependency rather than a dependency (that would be installed with the library?)